spec/dummy/.env.example

```diff
@@ -15,3 +15,13 @@ ADDRESS_FACADE_KEY = "client1"
 
 AIRBRAKE_HOST = "http://somewhere"
 AIRBRAKE_PROJECT_KEY ="<some key - use anything for dev and test>"
+
+DEVISE_MAILER_SENDER= "No reply <no-reply@environment-agency.gov.uk>"
+
+# In development, use mailcatcher on localhost:1025 to intercept mail and
+# view it at http://localhost:1080. See https://mailcatcher.me
+EMAIL_USERNAME=''
+EMAIL_PASSWORD=''
+EMAIL_APP_DOMAIN=''
+EMAIL_HOST='localhost'
+EMAIL_PORT='1025'
```

Comment:
To be consistent with the rest of the code base, the strings should be defined with double quotes, not single. For example:

```
EMAIL_PORT="1025"
```